src/VirtualClient/VirtualClient.Main/profiles/PERF-IO-FIO-MULTITHROUGHPUT.json

```diff
@@ -1,89 +1,89 @@
-                "Tags": "IO,FIO,MultiThroughput,OLTP"
-            }
-        }
-    ],
-    "Dependencies": [
-        {
-            "Type": "FormatDisks",
-            "Parameters": {
-                "Scenario": "InitializeDisks"
-            }
-        },
-        {
-            "Type": "LinuxPackageInstallation",
-            "Parameters": {
-                "Scenario": "InstallRequiredLinuxPackages",
-                "Packages-Apt": "libaio1,libaio-dev",
-                "Packages-Dnf": "libaio,libaio-devel",
-                "Packages-Yum": "libaio,libaio-devel",
-                "Packages-Zypper": "libaio1,libaio-dev"
-            }
-        },
-        {
-            "Type": "DependencyPackageInstallation",
-            "Parameters": {
-                "Scenario": "InstallFIOWorkloadPackage",
-                "BlobContainer": "packages",
-                "BlobName": "fio.3.30.0.zip",
-                "PackageName": "fio",
-                "Extract": true
-            }
-        }
-    ]
+    {
+      "Type": "FioMultiThroughputExecutor",
+      "Parameters": {
+        "Scenario": "fio_multithroughput",
+        "PackageName": "fio",
+        "DirectIO": "1",
+        "DiskFilter": "$.Parameters.DiskFilter",
+        "DurationSec": "300",
+        "GroupReporting": "0",
+        "RandomIOFileSize": "$.Parameters.RandomIOFileSize",
+        "RandomReadBlockSize": "8K",
+        "RandomReadNumJobs": 1,
+        "RandomReadQueueDepth": 512,
+        "RandomReadWeight": 5416,
+        "RandomWriteBlockSize": "8K",
+        "RandomWriteNumJobs": 1,
+        "RandomWriteQueueDepth": 512,
+        "RandomWriteWeight": 4255,
+        "SequentialIOFileSize": "$.Parameters.SequentialIOFileSize",
+        "SequentialReadBlockSize": "56K",
+        "SequentialReadNumJobs": 1,
+        "SequentialReadQueueDepth": 64,
+        "SequentialReadWeight": 0,
+        "SequentialWriteBlockSize": "56K",
+        "SequentialWriteNumJobs": 1,
+        "SequentialWriteQueueDepth": 64,
+        "SequentialWriteWeight": 329,
+        "TargetIOPS": "$.Parameters.TargetIOPS",
+        "TargetPercents": "$.Parameters.TargetPercents",
+        "TemplateJobFile": "oltp-c.fio.jobfile",
+        "Tags": "IO,FIO,MultiThroughput,OLTP"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "FormatDisks",
+      "Parameters": {
+        "Scenario": "InitializeDisks"
+      }
+    },
+    {
+      "Type": "LinuxPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallRequiredLinuxPackages",
+        "Packages-Apt": "libaio1,libaio-dev",
+        "Packages-Dnf": "libaio,libaio-devel",
+        "Packages-Yum": "libaio,libaio-devel",
+        "Packages-Zypper": "libaio1,libaio-dev"
+      }
+    },
+    {
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallFIOWorkloadPackage",
+        "BlobContainer": "packages",
+        "BlobName": "fio.3.30.0-1.zip",
```

Comment:
@deep1712 , please note that I changed the public package to fio.3.30.0.zip, without the dash-1. This workload will work without a package access key. However to make this also work in our  private storage, I need to copy fio.3.30.0.zip back to our private storage.